releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -54,74 +56,101 @@ An initial Stellar chain config needs to be added to `${ENV}.json` file under `c
   }
 ```
 
-Deploy Stellar Gateway
-
-1. Deploy Gateway
+#### Mainnet:
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
+"stellar": {
+    "name": "Stellar",
+    "axelarId": "stellar",
+    "rpc": "https://mainnet.sorobanrpc.com",
+    "horizonRpc": "https://horizon.stellar.org",
+    "networkType": "mainnet",
+    "chainType": "stellar",
+    "decimals": 7,
+    "finality": "1",
+    "approxFinalityWaitTime": 1,
+    "tokenSymbol": "XLM",
+    "tokenAddress": "CAS3J7GYLGXMF6TDJBBYYSE3HQ6BBSMLNUQ34T6TZMYMW2EVH34XOWMA",
+    "explorer": {
+      "name": "Soroban Nownodes",
+      "url": "https://soroban.nownodes.io/mainnet"
+    },
+  "contracts": {}
+  }
 ```
 
-2. After deploying the Stellar Gateway, ensure that you [deploy CosmWasm for Stellar GMP v1.0.0](../cosmwasm/2025-01-Stellar-GMP-v1.0.0.md).
+1. Deploy Gateway
+
+| Network              | `minimumRotationDelay` | `nonce` | `previousSignersRetention` |
+| -------------------- | ---------------------- | ------- | -------------------------- |
+| **Devnet-amplifier** | `"0"`                  | `"0"`   | `"15"`                     |
+| **Stagenet**         | `"0"`                  | `"0"`   | `"15"`                     |
+| **Testnet**          | `"0"`                  | `"0"`   | `"15"`                     |
+| **Mainnet**          | `TBD`                  | `"0"`   | `TBD`                      |
 
-- Once all steps are completed, update ampd with the Stellar chain configuration, then follow the remaining steps.
+```bash
+node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0 --minimum-rotation-delay 0 --nonce 0 --previous-signers-retention 15
+```
 
-3. Rotate genesis verifier set on Stellar Gateway
+2. Rotate genesis verifier set on Stellar Gateway
 
 ```bash
-node stellar/gateway.js rotate --chain-name $CHAIN
+node stellar/gateway.js rotate
 ```
 
-4. Deploy Gas Service
+3. Deploy Gas Service
 
 ```bash
 node stellar/deploy-contract.js deploy axelar_gas_service --version v1.0.0
 ```
 
-5. Deploy Operators
+4. Deploy Operators
 
 ```bash
 node stellar/deploy-contract.js deploy axelar_operators --version v1.0.0
 ```
 
+5. Register refunder address as an operator
```

Comment:
Need to get the list of refunder addresses from the relayer team
```suggestion
5. Get the list of operators used by the relayer and register on the Operators contract.
```
Refunder is the wrong term, operators contract doesn't care what the target contract is. Any contract can be called by whitelisted operators